flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/data/TestFlinkParquetReaderWriter.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.flink.data;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Iterator;
+import org.apache.flink.table.data.RowData;
+import org.apache.flink.table.types.logical.LogicalType;
+import org.apache.iceberg.Files;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.data.DataTest;
+import org.apache.iceberg.data.RandomGenericData;
+import org.apache.iceberg.flink.FlinkSchemaUtil;
+import org.apache.iceberg.flink.TestHelpers;
+import org.apache.iceberg.io.CloseableIterable;
+import org.apache.iceberg.io.FileAppender;
+import org.apache.iceberg.parquet.Parquet;
+import org.junit.Assert;
+
+public class TestFlinkParquetReaderWriter extends DataTest {
```

Comment:
What is this doing that isn't already accomplished by `TestFlinkParquetReader` or `TestFlinkParquetWriter`?

Reply:
As mentioned above, when I read this part of code, I found that:

- The TODO describes that a unit test needs to be added to verify decimal type writer\read, but org.apache.iceberg.data.DataTest#testSimpleStruct seems contains this type. Is this enough?
- There is no unit test that uses both FlinkParquetWrite and FlinkParquetWriteReader.

Based on this, I submitted this PR. If this is not necessary or my understanding is incorrect, let me close it.

Reply:
Thanks for working on this, but I don't think that it is necessary. I think that we have plenty of coverage and better testing using the other two.